src/include/sof/audio/component.h

```diff
@@ -47,7 +47,6 @@ struct sof_ipc_stream_posn;
 struct dai_hw_params;
 
 /** \addtogroup component_api Component API
- *  Component API specification.
  *  @{
```

Comment:
Why are we removing this line?

Reply:
When you read the header, it is just repeated *Component API* from the line above, no value here. Breathe also does not use this additional text in any way. The group is just rendered inline in the place of reference, so it does not appear as longer former of "Component API" anywhere.

Reply:
@mmaka1 this is the kind of information we need in the commit message. sorry for troubling you with this.